dev/ssl_darwin.go

```diff
@@ -3,13 +3,14 @@ package dev
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"os/exec"
 
 	"github.com/puma/puma-dev/homedir"
 )
 
-const supportDir = "~/Library/Application Support/io.puma.dev"
+const SupportDir = "~/Library/Application Support/io.puma.dev"
```

Comment:
export this so it can be used in tests for `main`. Being able to have other packages do `dev.SupportDir` seems totally appropriate.